File: Pranav_Agrahari/Basics/q3.java
import java.util.Scanner;

public class q3 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter your radius ");
        int r  = sc.nextInt();
        double area = (3.14*(r*r));
        double circumference = 2*3.14*r;
        System.out.println("The area of the circle is " + area);
        System.out.println("The circumference of circle is " +circumference);

    }
}

File: isha/assignment1/Q3.cpp
//WAP to calculate and print area of circle and circumference of circle.
#include <iostream>
using namespace std;

const double PI = 3.14159;

double area(double radius) {
    return PI * radius * radius;
}

double circumference(double radius) {
    return 2 * PI * radius;
}

int main() {
    double radius;
    cin >> radius;
    cout << "Area: " << area(radius) << endl;
    cout << "Circumference: " << circumference(radius) << endl;
    return 0;
}
